examples/examples/z_ping.rs

```diff
@@ -16,6 +16,7 @@ use std::time::{Duration, Instant};
 use zenoh::config::Config;
 use zenoh::prelude::sync::*;
 use zenoh::publication::CongestionControl;
+use zenoh::sample_builder::QoSBuilderTrait;
```

Comment:
I suggest to add the builder traits directly in the prelude.

Reply:
done